docs/install/index.md

```diff
@@ -60,6 +60,16 @@ For Cuda 12.x, please use the following instructions to install from [ORT Azure
 pip install onnxruntime-gpu --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
 ```
 
+#### Install ONNX Runtime GPU (ROCm)
+For ROCm, please follow instructions to install it at the [AMD ROCm install docs](https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.0.0/). The ROCm execution provider for ONNX Runtime is built and tested with ROCm 6.0.0
+
+To build on Linux, use the following instructions: 
+```bash
+./build.sh --config <Release|RelWithDebInfo> --use_rocm --rocm_home <path_to_ROCm_home>
+```
+
+See more information about the ROCm Execution Provider [here](https://onnxruntime.ai/docs/execution-providers/ROCm-ExecutionProvider.html).
```

Comment:
Is it better to add a link to https://onnxruntime.ai/docs/build/eps.html#amd-rocm for build instruction for Linux, instead of duplicating the content here?